project_resstock_national.yml

```diff
@@ -1,15 +1,29 @@
-schema_version: 0.2
-stock_type: residential
+schema_version: '0.3'
 buildstock_directory: ../resstock  # Relative to this file or absolute
 project_directory: project_national  # Relative to buildstock_directory
-output_directory: ../national_test_outputs
-weather_files_url: https://data.nrel.gov/system/files/128/ResStock_TMY3.zip
-# weather_files_path: ../ResStock_TMY3.zip  # Relative to this file or absolute path to zipped weather files
+output_directory: /scratch/nmerket/national_test_outputs2
+# weather_files_url: https://data.nrel.gov/system/files/128/ResStock_TMY3.zip
```

Comment:
Update url to https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip to be compatible with resstock 2.4.0.